acai-ios/CollectionViewCells/RadioSelectionCollectionViewCell.swift

```diff
@@ -0,0 +1,102 @@
+//
+//  RadioSelectionCollectionViewCell.swift
+//  acai-ios
+//
+//  Created by Artesia Ko on 3/17/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+
+class RadioSelectionCollectionViewCell: UICollectionViewCell {
+    
+    // MARK: View vars
+    
+    var titleLabel: UILabel!
+    var priceLabel: UILabel!
+    var imageView: UIImageView!
+    var radioCircle: UIView!
+    var radioFill: UIView!
+    var line: UIView!
+    
+    // MARK: Data
+    
+    var customizationOption: CustomizationOption!
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        imageView = UIImageView()
+        imageView.image = UIImage()
+        imageView.contentMode = .scaleAspectFit
+        self.addSubview(imageView)
+        imageView.snp.makeConstraints { (make) in
+            make.leading.equalToSuperview().offset(24)
+            make.height.width.equalTo(55)
+            make.centerY.equalToSuperview()
+        }
```

Comment:
for every subview, there should be a newline before the init and newline before you set the constraints